lib/DataFileUtil/DataFileUtilImpl.py

```diff
@@ -377,13 +377,7 @@ def _retrieve_filepath(self, file_url, cookies=None):
 
         """
 
-        try:
```

Comment:
You can mix try and with statements: https://stackoverflow.com/questions/713794/catching-an-exception-while-using-a-python-with-statement

IMO we shouldn't change the API unless we have to.

Reply:
👍 copy that.